libraries/adaptive-expressions/src/expressionEvaluator.ts

```diff
@@ -14,7 +14,7 @@ import { ReturnType } from './returnType';
  * Delegate for doing static validation on an expression.
  * Validators can and should throw exceptions if the expression is not valid.
  */
-export type ValidateExpressionDelegate = (expression: Expression) => any;
+export type ValidateExpressionDelegate = (expression: any) => any;
```

Comment:
>any [](start = 54, length = 3)

Why did all of these change from Expression to any?  It must be an Expression, right?

Reply:
right, this should be an Expression. Fixed.